.github/workflows/test-app.yml

```diff
@@ -0,0 +1,36 @@
+name: Test App
```

Comment:
Shouldn't this be something like `App Unit Tests`?

Reply:
Sounds good